README.md

```diff
@@ -1,15 +1,27 @@
 
 # Decidim Documentation 
+**[WORK IN PROGRESS]**
 
-This is the new repository for the [Decidim Documentation website](https://antora.org/):
+**Source code** of the [Decidim Documentation website](https://decidim-docs.netlify.app/en/decidim/).
 
-Is built with [Antora](https://antora.org/), although for this version we're going to try to not have multi-repositories, as it's a hassle to work with. The idea is to have all the documents living on this same repository. 
+**Newcomers** will find a [getting started](#decidim-documentation) section, with different [tutorials](#decidim-documentation) to learn decidim basics.  
+**Detailed information** are available in  [administrator](#decidim-documentation)/[developper](#decidim-documentation)/[deployement](#decidim-documentation) guides.
 
-This project is on active development and it's not on production yet.
+**As a community**, we have [social contract](#decidim-documentation) and [contribution guideline](#decidim-documentation) to use or participate to decidim. There is a [wide number of way](#decidim-documentation) to give an appreciable help !  
+Feel free to join us on [gitter](https://gitter.im/decidim/decidim).
 
-For working with it, you can do it several ways: 
+_Documentation built with [Antora](https://antora.org), with XYZ licence._
 
-## Install locally 
+**Index**
```

Comment:
I add an index, not totally convince about it, but it allow a quick overview of README content (like roadmap). It could make navigation easier.